fixtures/tracks/animal/symlinks/include-in-fish.txt

```diff
@@ -0,0 +1 @@
+This should get included in fish.
```

Comment:
You could maybe call it `.meta/shared` in the track repository, that's completely up to you.
Having it be just `.meta` in the test seems reasonable.